media-gfx/sane_backends/sane_backends-1.0.27.recipe

```diff
@@ -90,13 +92,13 @@ BUILD()
 		--libdir=$libDir \
 		--includedir=$includeDir \
 		--datadir=$dataDir \
-		--with-docdir=$docDir \
+		--docdir=$docDir \
```

Comment:
Any reason why these are needed? Running "runConfigure" should take care of all the above?

Reply:
Feel free to edit as needed. Eveything is tested as working.